ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -159,6 +159,44 @@ objects:
         mkdir -p /tmp/output
         cd /tmp/output
 
+        function retry() {
+          local ATTEMPTS="${1}"
+          shift
+          echo "${@}"
+          rc=1
+          for i in $(seq 0 ${ATTEMPTS}); do
+            "${@}" && rc=0 && break
+            sleep 10
+          done
+          if [ "${rc}" != "0" ]; then exit 1; fi
+        }
+
+        function setup_ssh_bastion() {
+          echo "Setting up ssh bastion"
+          mkdir -p ~/.ssh || true
+          cp "${KUBE_SSH_KEY_PATH}" ~/.ssh/id_rsa
+          chmod 0600 ~/.ssh/id_rsa
+          if ! whoami &> /dev/null; then
+            if [ -w /etc/passwd ]; then
+              echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:${HOME}:/sbin/nologin" >> /etc/passwd
+            fi
+          fi
+          curl https://raw.githubusercontent.com/eparis/ssh-bastion/master/deploy/deploy.sh | bash
+          for i in $(seq 0 60)
+          do
+            BASTION_HOST=$(oc get service -n openshift-ssh-bastion ssh-bastion -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
+            if [ ! -z "${BASTION_HOST}" ]; then break; fi
+            sleep 10
+          done
+        }
+
+        function bastion_ssh() {
+          retry 60 \
+            ssh -o LogLevel=error -o ConnectionAttempts=100 -o ConnectTimeout=5 -o StrictHostKeyChecking=no \
+                -o ProxyCommand="ssh -A -o StrictHostKeyChecking=no -o LogLevel=error -o ServerAliveInterval=30 -o ConnectionAttempts=100 -o ConnectTimeout=5 -W %h:%p core@${BASTION_HOST}" \
+                $@
```

Comment:
This should be quoted.  Checking:

```console
$ function testing() {
>   python -c 'import sys; print(sys.argv)' $@
> }
$ testing 'hello, world'
['-c', 'hello,', 'world']
$ function testing() {
>   python -c 'import sys; print(sys.argv)' "$@"
> }
$ testing 'hello, world'
['-c', 'hello, world']
```

Reply:
Quoting this in `retry`, otherwise bastion_ssh won't retry